coa-helm/templates/cbas-api.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.cbas.coaEnabled }}
+{{- if .Values.cbas.enabled }}
```

Comment:
Not sure if a corresponding Leo PR is coming up, but might want to change these values? https://github.com/DataBiosphere/leonardo/pull/3128/files#r1103089975

Reply:
Yup! I did that locally - going to update the generated PR when it comes